core/api/src/graphql/public/root/mutation/ln-invoice-cancel.ts

```diff
@@ -0,0 +1,54 @@
+import dedent from "dedent"
+
+import { Wallets } from "@/app"
+
+import { GT } from "@/graphql/index"
+import { mapAndParseErrorForGqlResponse } from "@/graphql/error-map"
+
+import WalletId from "@/graphql/shared/types/scalar/wallet-id"
+import PaymentHash from "@/graphql/shared/types/scalar/payment-hash"
+import SuccessPayload from "@/graphql/shared/types/payload/success-payload"
+
+const LnInvoiceCancelInput = GT.Input({
+  name: "LnInvoiceCancelInput",
+  fields: () => ({
+    walletId: {
+      type: GT.NonNull(WalletId),
+      description: "Wallet ID for a BTC wallet belonging to the current account.",
```

Comment:
Doesn't this work for stablesats invoices too?

Reply:
Updated https://github.com/GaloyMoney/galoy/pull/4107/commits/c4221b7a1c0724ea45d5856def3c3101777947e9